cocos/scripting/js-bindings/manual/JavaScriptJavaBridge.cpp

```diff
@@ -508,6 +508,10 @@ static bool JavaScriptJavaBridge_callStaticMethod(se::State& s)
         {
             int count = argc - 3;
             jvalue* jargs = new jvalue[count];
+            int *toRelease = new int[count];
```

Comment:
Where to delete[] toRelease array?
Would using `std::vector` be better?

Reply:
Yes you are right. I Will modify a bit to meet this comment.